pythonwhois/net.py

```diff
@@ -95,9 +104,10 @@ def get_root_server(domain):
 	raise shared.WhoisException("No root WHOIS server found for domain.")
 
 def whois_request(domain, server, port=43):
+    format = WHOIS_REQUEST_FORMAT_EXCEPTIONS.get(server, "%s\r\n")
```

Comment:
I believe `format` has a special meaning - perhaps it'd be better to name this something more descriptive like `formatted_request`?